messages/help.txt

```diff
@@ -1,5 +1,5 @@
 *ŠALÁT + MÄSO*
-stačí nahlásiť do ~11:00, že budete papať a dostanete šalátik + mäsko, aké si vypýtate z Billy. Bežne býva kuracie stehno, klobáska, no nájde sa občas aj sekaná, prso, bôčik, ...
+stačí nahlásiť do ~11:00, že budete papať a dostanete šalátik + mäsko, aké si vypýtate z Billy. Bežne býva kuracie stehno, klobáska, no nájde sa občas aj sekaná, prsia, bôčik, ...
```

Comment:
We don't buy meat in Billa anymore. Also currently only roasted chicken is prepared by our kitchen ninja.